### DISCUSSION TITLE
Glitch Found

### DISCUSSION THREAD

**Discussion by honorcoding** [code4recovery/12-step-meeting-list#1117] [General]
When editing an individual meeting, in the "Contact Information" section, there is a sub-section called "Contacts". There you can input names, emails and phone numbers of folks who are designated as the contact for that meeting. 

We discovered that when you have a phone number, but no contact name, then the meeting does not appear in the meeting listing on the front-end of the site.

We got around this by temporary filling the name field with the word "Contact". 

This appears to be a glitch. If it was intentional, then it is not a good idea, because it causes confusion. 

Just wanted to let folks know, so you can fix for the next update.

**Comment by kiyote33** [comment]
I don't think this is so much a glitch, but an unexpected way of posting a group contact phone number. By design, when a group wants a contact phone number to display for their meeting listing they simply fill in the Phone field. The image below show's the intended way to post a group's phone number along with the new work around you discovered. 
![image](https://user-images.githubusercontent.com/57125427/233470144-7584fcf2-0aa4-4cdd-a469-2511571771fe.png)
